src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -331,6 +334,74 @@ struct ScalarOp<ONNXTanOp> {
   using IOp = NotSuportedScalarOp;
 };
 
+//===----------------------------------------------------------------------===//
+// Scalar unary ops for lowering ONNXGeluOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXGeluOp> {
+  using FOp = CustomScalarOp;
+  using IOp = CustomScalarOp;
+};
+
+template <>
+double analyzeSimdFor<ONNXGeluOp>(
+    Type t, Operation *op, int64_t &von, int64_t &son) {
+  double results;
+  StringRef approximate = dyn_cast<ONNXGeluOp>(op).getApproximate();
+  if (approximate == "none")
```

Comment:
@AlexandreEichenberger Let me know what you think of the SIMD for Gelu... I think I added every operation but it is always good to have another set of eyes. Let me know if there is anything I am missing. Thanks :)